apps/chat/src/utils/server/api.ts

```diff
@@ -67,12 +83,29 @@ export const parseConversationApiKey = (
       ? [EMPTY_MODEL_ID, apiKey] // receive without prefix with model i.e. {name}
       : [decodeModelId(parts[0]), parts.slice(1).join(pathKeySeparator)]; // receive correct format {modelId}__{name}
 
-  return {
+  const parsedApiKey: Omit<ConversationInfo, 'folderId' | 'id'> = {
     model: { id: modelId },
     name,
     isPlayback: modelId === PseudoModel.Playback,
     isReplay: modelId === PseudoModel.Replay,
   };
+
+  if (options?.parseVersion) {
+    const version = parts.at(-1);
```

Comment:
```suggestion
    const version = parts.length > 2 && parts.at(-1);
```

Reply:
in this case NA_VERSION won't be handled